lib/virt_autotest/utils.pm

```diff
@@ -550,6 +551,12 @@ sub create_guest {
     my ($guest, $method) = @_;
     my $v_type = $guest->{name} =~ /HVM/ ? "-v" : "";
 
+    # Ensure UEFI firmware package is installed
+    if (script_run("rpm -q ovmf") != 0) {
+        record_info("Installing OVMF", "Installing OVMF package for UEFI support");
+        zypper_call("in ovmf");
```

Comment:
This is executed on host. Would you please explain why the host need ovmf? I remember yesterday you clarified that legacy boot machine will be fine to run the test.

Reply:
Besides, it runs unconditionally based on logic. It will not be needed for existing MU tests, right?